packages/block-editor/src/components/block-list/index.js

```diff
@@ -37,6 +37,9 @@ function BlockList( {
 	isDraggable,
 	renderAppender,
 	__experimentalUIParts = {},
+	tagName = 'div',
+	forwardedRef,
```

Comment:
I'm not sure I understand why we can't just use `React.forwardRef` and use just "ref" on the caller?

Reply:
Yes, sorry, we should do that :)